src/multi-selector.js

```diff
@@ -196,7 +196,7 @@ angular.module('cp-multi-selector')
 				 */
 				$('body').on('click.cpmultiselector' + id, function(e) {
 					if(!$(e.target).closest(el).length) {
-						scope.$apply(function() {
```

Comment:
This is the root of the cause. You can't call `scope.$apply` arbitrarily in a dom event handler. A digest might already be underway.